.github/workflows/tests.yml

```diff
@@ -9,12 +9,12 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        php: [7.4, 8.0]
-        laravel: [8.*]
+        php: [8.0]
+        laravel: [9.*]
         dependency-version: [prefer-stable]
         include:
```

Comment:
This `include` is probably unnecessary given it matches the current constraints.